tests/bunit.web.tests/JSInterop/InvocationHandlers/VirtualizeJSRuntimeInvocationHandlerTest.cs

```diff
@@ -11,17 +11,20 @@ namespace Bunit.JSInterop.ComponentSupport;
 
 public class VirtualizeJSRuntimeInvocationHandlerTest : TestContext
 {
-	public static readonly IEnumerable<object[]> ItemsInCollection = new object[][]
-	{
-			new object[] { 0 }, new object[] { 7 }, new object[] { 30 }, new object[] { 60 }, new object[] { 100 }, new object[] { 300 }, new object[] { 500 },
-	};
+	public static readonly IEnumerable<object[]> ItemsInCollection =
+	[
+		[0], [7], [30], [60], [100], [300], [500],
+	];
```

Comment:
```suggestion
	public static readonly TheoryData<int> ItemsInCollection = new TheoryData<int>()
	{
		0, 7, 30, 60, 100, 300, 500,	
	};
```

I think something like this should work.

Reply:
Done